src/test/java/jenkins/branch/WorkspaceLocatorImplTest.java

```diff
@@ -267,10 +268,10 @@ public void move() throws Exception {
     @Test
     public void collisions() throws Exception {
         WorkspaceLocatorImpl.MODE = WorkspaceLocatorImpl.Mode.ENABLED;
-        FilePath firstWs = r.buildAndAssertSuccess(r.createFreeStyleProject("first-project-with-a-rather-long-name")).getWorkspace();
-        assertEquals("-project-with-a-rather-long-name", firstWs.getName());
+        FilePath firstWs = r.buildAndAssertSuccess(r.createFreeStyleProject("first_project-with-a-rather-long-name")).getWorkspace();
+        assertEquals("_project-with-a-rather-long-name", firstWs.getName());
         firstWs.deleteRecursive();
-        assertEquals("roject-with-a-rather-long-name_2", r.buildAndAssertSuccess(r.createFreeStyleProject("second-project-with-a-rather-long-name")).getWorkspace().getName());
+        assertEquals("roject-with-a-rather-long-name_2", r.buildAndAssertSuccess(r.createFreeStyleProject("second_project-with-a-rather-long-name")).getWorkspace().getName());
```

Comment:
This change is unnecessary I think, but OK.

Reply:
true. but I thought it would be a little bit misleading to write
first_project-with-a-rather-long-name
but
second_project-with-a-rather-long-name

Reply:
```suggestion
        assertEquals("roject-with-a-rather-long-name_2", r.buildAndAssertSuccess(r.createFreeStyleProject("second-project-with-a-rather-long-name")).getWorkspace().getName());
```